recipes/flask-mqtt/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "flask-mqtt" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stlehmann/Flask-MQTT/releases/download/{{ version }}/Flask-MQTT-{{ version }}.tar.gz
+  sha256: d196c4a7a1fc37a4364e8fe46b3670cc656a851c5b63d69e2836c80e03da88f1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+  host:
```

Comment:
In general, only `pip` and `python` are enough for the `host` section. Can you please try that?